### ISSUE TITLE
Nebular line flux is not conserved outside the MILES window: 86/128 lines carrying 70% of sum(L) are under-resolved, recovery 0.003x-3.0x (GALEX NUV photometry +271%)

### ISSUE THREAD

**Issue by suchethac** [suchethac/tengri#1836]
## Summary

Nebular emission lines are rendered onto the SSP wavelength grid as triweight profiles normalized to **unit area in frequency** — analytically exact, but only recovered by the discrete integral if the grid resolves the profile. The FSPS/MILES grid has a high-resolution window at **3500–7500 Å (0.9 Å)** and is coarse everywhere else (8 Å → 29000 Å). Every line outside that window is under-resolved, and its integrated flux comes out wrong.

**86 of 128 Cue lines fall outside the MILES window. They carry 69.9 % of `sum(line_lums)`.** Measured recovery on the production grid ranges from **0.0026×** (99.7 % of the line lost) to **3.04×**.

Nothing warns. `line_lums` is published correctly; it is the *rendering into* `sed_nebular` that loses or invents flux, and photometry integrates that array.

All measurements on `fix-1738-nebular-single-component-docs` @ `82b37cd9d`, `fsps_prsc_miles_chabrier.h5`, Cue backend, 2026-08-16.

## Mechanism

`cue.py:1893-1899` builds the bucket as

```python
neb_sed = jnp.interp(ssp_wave, cont_wav, cont_lum, left=0.0, right=0.0)
neb_sed = neb_sed + render_nebular_lines(line_wav, line_lum, ssp_wave, ...)
```

`place_line_profiles_velocity` gives each line support half-width `h = 3σ_λ = 3(σ_v/c)λ`. At the default `neb_eline_sigma_kms = 100`, that is 1.2 Å at Lyα and 6.6 Å at Hα. The grid must sample `2h`; outside the MILES window it does not.

## Measurement 1 — refinement converges to `sum(L_l)`, the grid does not

Each line rendered alone, integrated in `dν`, divided by its published `line_lums`. Windowed refinement in log space:

| line (Å) | pts across `2h` | SSP grid | ×4 | ×16 | ×64 |
|---|---|---|---|---|---|
| **1215.7 (Lyα)** | 0.24 | **2.0742** | 0.6775 | 1.0061 | **1.0000** |
| 6564.6 (Hα) | 14.59 | 0.9997 | 1.0000 | 1.0000 | 1.0000 |
| 3729.9 [OII] | 8.29 | 1.0053 | 1.0000 | 1.0000 | 1.0000 |
| 5008.2 [OIII] | 11.13 | 1.0004 | 1.0000 | 1.0000 | 1.0000 |
| 4862.7 (Hβ) | 10.81 | 1.0001 | 1.0000 | 1.0000 | 1.0000 |
| **9533.2 [SIII]** | 0.95 | **1.5714** | 0.9996 | 1.0000 | 1.0000 |
| **334704** | 0.22 | **1.5693** | 0.3811 | 0.9977 | 1.0000 |
| **883323** | 0.22 | **0.0026** | 1.8511 | 0.9924 | 1.0000 |
| **518004** | 0.22 | **0.0636** | 0.3849 | 0.9973 | 1.0000 |

The ×64 column is **1.0000 for every line**. That is the control: the profile normalization is right and the target is right; only the sampling is wrong. The rule is `pts across 2h ≳ 4`.

Whole line block on the production grid: `∫rendered dν = 2.905e42` against `sum(L_l) = 1.292e42` — **2.2489×**.

## Measurement 2 — which lines are at risk

```
SSP grid spacing profile (fsps_prsc_miles_chabrier):
        91 -      1000 A : n=  131  median dlam=     8.000 A
      1000 -      3000 A : n=  200  median dlam=    10.000 A
      3000 -      3500 A : n=   25  median dlam=    20.000 A
      3500 -      7500 A : n= 4423  median dlam=     0.900 A   <- MILES window
      7500 -      9000 A : n=   75  median dlam=    20.000 A
      9000 -     20000 A : n=  210  median dlam=    50.000 A
     20000 -    100000 A : n=  370  median dlam=   200.000 A
    100000 - 100000000 A : n=  750  median dlam= 29000.000 A

lines with <4 pts across support: 86/128  (they carry 69.9% of sum(L))
  of those, inside the MILES window: 0
lines fully inside MILES window: 42, all resolved: True
```

The split is exactly the MILES window. Inside it the rendering is correct; outside it, never.

## Measurement 3 — it reaches broadband photometry

`bias = (dense-rendered line block − flux-exact line block) / total band L_nu`, same model, dust `tau_bc=tau_diff=1`:

| config | band | line frac of band | **photometric bias** |
|---|---|---|---|
| z=0.5, GALEX+SDSS | `galex_nuv` | 407 % | **+271 %** |
| z=0.5, GALEX+SDSS | `galex_fuv` | 10.6 % | **−11.7 %** |
| z=0.3, SDSS | `sdss_u` | 12.2 % | **+3.08 %** |
| z=1.0, SDSS | `sdss_r` | 3.1 % | **+1.30 %** |
| z=0.5, SDSS | `sdss_u` | 8.7 % | +1.12 % |
| z=0.05, SDSS | `sdss_z` | 0.9 % | +0.39 % |
| z=0.05, SDSS | `sdss_g/r/i` | 0.8–10.6 % | ≤ 0.02 % |

The GALEX NUV case is Lyα at 1824 Å observed — no IGM involvement at z=0.5, so that number stands on its own. The bias grows without bound wherever Lyα enters a band (`sdss_u` at z=2, `sdss_g` at z=3 both read >3000 %, though those bands also want IGM treatment my probe did not model, so I quote them only as "diverges", not as calibrated numbers).

**`sdss_griz` at z≈0.05 is clean to ≤0.4 %** — which is why every existing fixture misses this.

## Controls (each rules out an alternative explanation)

- **Units are right.** Every *resolved* line recovers 0.999–1.005 with the identical call. A unit error would move all 128 by one common factor.
- **Not the width floor.** `place_line_profiles_velocity` floors `h` at half the *median* grid spacing. 0/128 lines are floored — the median (0.9 Å) is set by the 4423 MILES points and is far below the local spacing where the failures occur. The floor is measuring the wrong thing.
- **Not filter-wing sampling.** Profile-averaged vs point-sampled filter transmission agree to `ratio = 1.000` for every dominant line at z ∈ {0.5, 1, 2}.
- **Not my standalone render.** The pipeline's own `state.derived["sed_nebular"]`, baseline-subtracted around isolated lines, gives Hβ 0.9999 and [OIII] 0.9989 but **Lyα 3.0432**, **[SIII] 9533 2.3404**, **334704 Å 0.0000**. (Blended pairs — the [OII] doublet, Hα/[NII] — are windowing artifacts of that check and are not quoted.)

## Scope

Affects the **exact** forward path, so it is not a precompute issue and does not go away with `approx=None`. Any backend that reaches `render_nebular_lines` is in scope (Cue measured; `cloudy_grid` / `cb19` / `mappings` share the renderer). It is worst for UV lines (Lyα, C IV, Mg II, [OII] 3727 at z where it leaves the window) and for NIR/IR lines ([SIII] 9533, Paschen, the mid-IR fine-structure set).

Open question I did not measure: when the fast nebular grid **is** armed (dust-free models, #1281), photometry comes from the per-Q_H table rather than this array. If that table is flux-exact, `WavePrecomp` would be *more* accurate than the exact path in that configuration — worth checking, and directly relevant to #1738's channel-by-channel bounds.

## Suggested directions (not a decision)

1. **Project lines analytically.** A line is 1–13 Å inside a 1000–2000 Å filter, so `sum_l L_l · λ_l · T_filt(λ_l(1+z)) / c / norm` is exact to `O((h/Δλ_filt)²) ≈ 1e-5` and needs no grid at all. Measured **2.76× cheaper** in gradient FLOPs than the dense band projection (1,323,596 vs 3,657,652). This changes what the photometry path *means*, though — it would no longer be "integrate the SED array".
2. **Refine the grid locally** around each line at build time. Preserves "photometry integrates the SED", costs grid points and touches every downstream consumer.
3. **Renormalize each rendered profile** to its discrete integral. Cheapest and most local, but a line spanning <1 grid point still cannot be placed correctly in wavelength — it fixes the flux and not the position.

Direction (1) is the only one that is also correct for spectroscopy at arbitrary resolution, but it is the largest change.

## Reproduction

```python
import numpy as np
from tengri import FIXED, Fixed, Observation, Photometry, SEDModel
from tengri.components.nebular._shared import render_nebular_lines
from tengri.components.stellar.sps.dsps_wrapper import load_ssp_data

model = SEDModel.build(
    ssp_data=load_ssp_data("data/fsps_prsc_miles_chabrier.h5"),
    observation=Observation(photometry=Photometry.from_names(["sdss_g"])),
    sfh={"type": "delayed", "all_params": FIXED, "log_total_mass": 10.0,
         "tau_gyr": 1.0, "age_gyr": 5.0},
    dust={"type": "none"}, neb={"type": "cue", "all_params": FIXED},
    redshift=Fixed(0.05),
)
st = model.predict_state({})
w = np.asarray(st.wave)
lw = np.asarray(st.derived["line_waves"])
ll = np.asarray(st.derived["line_lums"])
rendered = np.asarray(render_nebular_lines(lw, ll, w, 0.0, 100.0))
nu = 2.99792458e10 / (w * 1e-8)
print(abs(np.trapezoid(rendered, nu)) / ll.sum())   # -> 2.2489, want 1.0
```
